client/packages/system/src/Clinician/ClinicianSearchInput.tsx

```diff
@@ -0,0 +1,50 @@
+import { Autocomplete } from '@openmsupply-client/common';
+import { ClinicianFragment, useClinicians } from 'packages/programs/src';
```

Comment:
The imports from `packages/...` cause problems when running jest. Can you update please?

```suggestion
import { ClinicianFragment, useClinicians } from '@openmsupply-client/programs';
```